data/groups.yaml

```diff
@@ -172,22 +174,11 @@ postgresql:
     exception: https://pagure.io/fesco/issue/2217
     packages:
     - postgresql
-pypy:
```

Comment:
pypy is not ported, it is just that currently, all the builds were done with pypy. But when bootstrapping, they are done with python2. Not sure if we want to keep this here or not, just saying how it is.

Reply:
I'll put it back.It's not something portingdb tracks well, but I guess it deserves a comment.